crates/sui-framework/packages/sui-system/sources/sui_system_state_inner.move

```diff
@@ -1073,7 +1073,6 @@ module sui_system::sui_system_state_inner {
         }
     }
 
-    #[test_only]
```

Comment:
there a few functions in this package that we can directly use by adding the friendship